component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/SpaceMemberDAO.java

```diff
@@ -29,7 +29,9 @@
 public interface SpaceMemberDAO extends GenericDAO<SpaceMemberEntity, Long> {
     void deleteBySpace(SpaceEntity entity);
 
-    SpaceMemberEntity getSpaceMemberShip(String remoteId, Long spaceId, SpaceMemberEntity.Status status);
+    SpaceMemberEntity getSpaceMemberShip(String remoteId, Long spaceId, Status status);
+
+    List<Long> getSpaceIdentityIdsByUserRole(String userId, Status status, int offset, int limit);
```

Comment:
please use `username` or `userIdentityId` to avoid confusion when reading API or method signature